tests/robottelo/test_settings.py

```diff
@@ -0,0 +1,81 @@
+"""Unit tests for sample json settings"""
+import json
+import os
+
+from unittest2 import TestCase
+
+JSON_INVALID_ITEMS = ['hostname ', ' scheme', 'None', '']
```

Comment:
ahhh ok, just wanted to make sure that we couldn't just pick some random things from the list of valid items and inject invalid characters into them.